kamon-akka-http/src/main/scala/kamon/akka/http/AkkaHttp.scala

```diff
@@ -61,6 +61,9 @@ object AkkaHttp {
     }
   }
 
+  val addHttpStatusCodeAsMetricTag: Boolean =
```

Comment:
This value should follow the same pattern as the `nameGenerator` and updated on the `onReconfigure` hook.